docs/articles/documentation/templates/authentication/role-options.md

```diff
@@ -0,0 +1,33 @@
+### options.preserveUrl
+
+Use this option to control which page is opened after you switch to the role.
+
+By default, TestCafe navigates back to the page that was opened previously to switching to the role.
+Set the `preserveUrl` option to `true` to save the URL to which the browser was redirected after logging in.
```

Comment:
Can we say:
To save the browser's redirect URL (after login), set the `preserveUrl` option to `true`.

Reply:
I wouldn't understand what `browser's redirect URL (after login)` means...

I'd rather go with something like that:

*To save the URL where the browser redirects after login, set the preserveUrl option to true.*

Reply:
Can we say:
To save the URL where a browser is redirected after login, set the preserveUrl option to true.